lib/livebook_web/components/core_components.ex

```diff
@@ -2,6 +2,7 @@ defmodule LivebookWeb.CoreComponents do
   use Phoenix.Component
 
   alias Phoenix.LiveView.JS
+  import LivebookWeb.Gettext
```

Comment:
We don't need to use gettext. You can remove this module and remove the usage of gettext from the table. :)